<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>261</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Поисковые фразы </string>
    </property>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_finish">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plain_text_edit_phrazes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Экспорт в excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clean">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_emails">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
